README.md

```diff
@@ -1,74 +1,20 @@
-# Gene & functional consequence mapping pipeline
+# Consequence prediction pipelines for ClinVar/Open Targets
 
-## Outline
+This repository contains two modules and their corresponding wrapper scripts:
+* [vep_mapping_pipeline](vep_mapping_pipeline): maps variants (CHROM:POS:REF:ALT) to their most severe functional
+  consequence according to Ensembl VEP, as well as their Ensembl gene ID and name.
+* [repeat_expansion_variants](repeat_expansion_variants): parses ClinVar `variant_summary` file and extracts
+  information about repeat expansion variants.
 
-The pipeline located in the [vep_mapping_pipeline](/vep_mapping_pipeline) directory performs gene & functional consequence mapping. That is, it takes variants as input and maps each of them to a gene (Ensembl gene ID & gene name) and to the most severe functional consequence which a variant causes in that gene, according to Variant Effect Predictor.
+Please see the corresponding module README file for more information.
 
-## Running the pipeline, input & output formats
+## Installing requirements
 
-### Input file
-The pipeline uses succinct, VCF-compatible variant identifiers, in the format of `CHROM:POS:REF:ALT`. Input file must contain variants in exactly this format, one entry per line. Example:
-```
-10:110821920:G:A
-10:110821933:C:T
-10:110821934:G:A
-```
+The commands below has been tested for Ubuntu 19.10. You might have to adjust commands and package names if you're using
```

Comment:
Wait, the travis YML says "bionic" which is Ubuntu 18.04...??

Reply:
Haha, fair point. Sorry for the confusion. I did initially test everything on Ubuntu 19.10, which my desktop is running. Then I wrote Travis config and it turned out that the latest version available is bionic, 18.04. Luckily it already includes decent package versions and everything works on it as well.

I have changed wording to "tested for Ubuntu 18.04 and newer". Fixed in https://github.com/EBIvariation/vep-mapping-pipeline/pull/4/commits/02e91604f830591c43d9c13404a938f328aa8f09